app/views/bankdetails/summary.scala.html

```diff
@@ -100,6 +101,10 @@
         }
     }
 
+    @form(EmptyForm, controllers.bankdetails.routes.SummaryController.post(index)) {
+        @submit(false, Some(Messages("button.checkyouranswers.acceptandaddbankaccount")), false)
+    }
+
     @anchor(
```

Comment:
Should this anchor come out? Will it not leave a second button on the page that looks very similar to the form button?

Reply:
yes...